.github/workflows/test.yaml

```diff
@@ -34,7 +40,7 @@ jobs:
         ci_node_index: [0, 1, 2]
     env:
       ALLOW_ANONYMOUS_LOGIN: "yes"
-      CONFDIR: "/app/samvera/hyrax-webapp/solr/config"
```

Comment:
utk-hyku and newer hyku/hyrax use `CONFDIR: "/app/samvera/hyrax-webapp/solr/conf` so this needs to be dynamically passed from the caller workflow